### PR TITLE
Comment Review

### PR DESCRIPTION
**Pull request by TomMaullin**
This PR addresses issue #14 by ensuring all comments have a similar format and do not include details of any functionality not currently included in the SwE code.

### CODE REVIEW

**Comment by nicholst** [comment]
I only just started to take a peak, but it's looks great. Oh!  One thing:  Don't mess with swe_DesRep.m beyond maybe a first line or something "This files is a clone of spm_DesRep.m, temporarily added to SWE until changes are made to the original SPM version"  ... i.e. once SPM fixes the LaTeX thing we can revert to using spm_DesRep, so there's no point to all the changes there (and, in fact, they are confusing, as it suggests we have full support for covariance model surfing, etc, which we don't).

**Comment by TomMaullin** [comment]
Hi @nicholst ,

Ah okay! I have reverted the changes I had made to `swe_DesRep.m` and written only a few lines at the top as you have suggested. I have now finished working on all other files and this PR is now ready for review.

File: swe_cfg_batch.m [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+	    'This is the batch interface for SwE, i.e. Sandwich Estimator '...
+	    'Method for Neuroimaging Longitudinal Data Analysis.']
+	                  }';
+	swe.values  = {swe_cfg_design};
+	              
+	%----------------------------------------------------------------------
 
-return
\ No newline at end of file
+end
```
Comment (nicholst) [inline swe_cfg_batch.m]: add newline

**Comment by nicholst** [comment]
See minor comments above.  Major:  There is a strong Matlab convention that functions have their documentation **after** the `function y = fun(x)` line (see [here](https://www.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html)).  You've put the function line after all of your comments. 

Also, as a small thing, Matlab help has a formal convention that the *first* line of the help is a single-line description of the function/file.  Your 'first' line is usually a horizontal rule of ===='s.

So!  Please revise so that (1) comments come after the function line and (2) replace the first horizontal rule with a single-line description of what the file/function does.

File: swe_DesRep.m [hunk 2]
```diff
@@ -256,11 +260,11 @@
 %-Format arguments
 %--------------------------------------------------------------------------
 if ~nargin
-    SPMid = spm('FnBanner',mfilename,SVNid);
+    SWEid = spm('FnBanner',mfilename,SVNid);
```
[the remark this answers is not in the document]
Comment (TomMaullin) [reply swe_DesRep.m]: Ah apologies! Yes, sorry I had done this before your previous comment and forgot about it when reverting - I will fix this now.

**Comment by TomMaullin** [comment]
Hi @nicholst , 

I believe I have now addressed all comments - please let me know if you have any further feedback!

File: swe_getSPM.m [hunk 3]
```diff
@@ -1,41 +1,48 @@
 function [SwE,xSwE] = swe_getSPM(varargin)
-% Compute a specified and thresholded SwE for the SwE method
+% Compute a specified and thresholded SwE parametric map for
```
Comment (nicholst) [inline swe_getSPM.m]: Make 1 line please

**Comment by TomMaullin** [comment]
Hi @nicholst ,

Ah sorry! I must have misread when you said `single-line` - these should now all be updated.

**Comment by nicholst** [comment]
Almost there!

Can you make it so there is consistency over all files w.r.t. to newlines and the horizontal rules (=====).

Right now, sometimes there is a blank link after the 1-line summary, sometimes a space and then text, or sometimes the rule (=====) right afterwards.

Can you make it so it's all the same, e.g. 1-line summary and then ===== ?

**Comment by TomMaullin** [comment]
This should now be addressed! Please let me know if you have any further feedback!

**Pull request merged**